shell/components/ResourceDetail/Masthead.vue

```diff
@@ -504,10 +504,32 @@ export default {
               {{ namespace }}
             </span>
           </span>
-          <span v-if="parent.showAge">{{ t("resourceDetail.masthead.age") }}: <LiveDate
-            class="live-date"
-            :value="value.creationTimestamp"
-          /></span>
+          <span v-if="parent.showAge">
+            {{ t("resourceDetail.masthead.age") }}:
+            <LiveDate
+              class="live-date"
+              :value="value.creationTimestamp"
+            />
+          </span>
+          <span
+            v-if="value.showCreatedBy"
+            data-testid="masthead-subheader-createdBy"
+          >
+            {{ t("resourceDetail.masthead.createdBy") }}:
+            <router-link
+              v-if="value.createdBy.location"
+              :to="value.createdBy.location"
+              data-testid="masthead-subheader-createdBy_link"
```

Comment:
very nitpicky, I know, but shouldn't we do `masthead-subheader-createdBy-link` instead of `masthead-subheader-createdBy_link`? 

It looks cleaner, imo. Same for the the `data-testid` below this one... By any means this is a blocker so ping me if this is the only item left to address and you need this to be moved forward